package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "virtual-keyboard",
+  "version": "1.0.0",
+  "description": "Virtual keyboard",
+  "main": "index.js",
+  "scripts": {
+    "lint": "npx eslint .",
```

Comment:
So, we have 20 problems here: 

![image](https://user-images.githubusercontent.com/3214353/79078827-cd462b80-7d13-11ea-9344-120f39ef3369.png)

There is no chance your code will pass a pipeline with errors. Please determine why they are marking your code as broken, and either fix code or adjust a config.

Reply:
Done